src/DTS/eBaySDK/Types/BaseType.php

```diff
@@ -553,12 +727,27 @@ private static function encodeValueXml($value)
         }
         else if (is_bool($value)){
             return $value ? 'true' : 'false';
+        }
+        else if (is_string($value)){
```

Comment:
This causes the tests to fail with the following errors.

DTS\eBaySDK\Types\Test\AmountClassTest::testToXml
Failed asserting that two DOM documents are equal.

DTS\eBaySDK\Types\Test\ComplexClassTest::testToXml
Failed asserting that two DOM documents are equal.